ui/app/components/ui/list-item/index.scss

```
@@ -1,14 +1,13 @@
 .list-item {
   width: 100%;
-  min-width: 374px;
   min-height: 86px;
-  margin: 0 20px;
+  margin: 0px;
```

Nit: The unit can be omitted for zero

```suggestion
  margin: 0;
```

We do have a stylelint rule for this, but our stylelint config is broken at the moment.